src/gvpn_controller.py

```diff
@@ -24,17 +25,18 @@ def __init__(self, user, password, host, ip4):
                 self.uid_ip_table[uid] = ip
 
         if CONFIG["icc"]:
-            self.inter_controller_conn()
+            self.inter_controller_conn() # UDP Server for Inter Controller Connection.
```

Comment:
If it's not a sentence, don't put a period.

Additionally, expanding "Inter Controller Connection" is redundant, because that's what the method name already says. Instead, a comment like `# UDP` server would suffice.